### PR TITLE
Task 4 testing

### PR DESCRIPTION
**Pull request by gegorov**
- [x] Coverage > 60%
- [x] Used snapshot testing
- [x] Coverage > 80%. Used enzym's shallow, mount and render 
- [x] Used cypress for e2e

### CODE REVIEW

File: cypress/integration/netflix-roulette/movie-app.spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+describe('NetflixRoulette', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000');
+  });
+
+  it('finds H1', () => {
```
Comment (rkhaslarov) [inline cypress/integration/netflix-roulette/movie-app.spec.js:6]: it's not a function, a test should be started with "should" word and then scenario

File: src/components/Footer/Footer.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Footer from './Footer';
+
+describe('Footer', () => {
+  it('to match snapshot', () => {
```
Comment (rkhaslarov) [inline src/components/Footer/Footer.spec.tsx:7]: it's not a function, a test should be started with "should" word and then scenario

File: src/components/Logo/Logo.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from 'enzyme';
+
+import Logo from './Logo';
+
+describe('Logo', () => {
+  it('Should render text', () => {
```
Comment (rkhaslarov) [inline src/components/Logo/Logo.spec.tsx:7]: lower case

File: cypress/integration/netflix-roulette/movie-app.spec.js (hunk 1 above)
Comment (gegorov) [reply cypress/integration/netflix-roulette/movie-app.spec.js]: fixed

File: src/components/Footer/Footer.spec.tsx (hunk 2 above)
Comment (gegorov) [reply src/components/Footer/Footer.spec.tsx]: fixed

File: src/components/Logo/Logo.spec.tsx (hunk 3 above)
Comment (gegorov) [reply src/components/Logo/Logo.spec.tsx]: fixed

**Pull request merged**